### ISSUE TITLE
App won't start with quarkus 2.5.1.Final + quarkus-logging-logback, it causes an application startup error

### ISSUE THREAD

**Issue by eriknyk** [quarkiverse/quarkus-logging-logback#22]
Using quarkus version: 

```
implementation 'io.quarkiverse.logging.logback:quarkus-logging-logback:0.5.0'
```

```
quarkusPluginVersion=2.5.1.Final
quarkusPlatformVersion=2.5.1.Final
```
It causes an application startup error: `LoggingLogbackProcessor#removeSlf4jBinding threw an exception: java.lang.NoSuchMethodError: io.quarkus.deployment.builditem.RemovedResourceBuildItem.`

### STACKTRACE

```
2021-12-06 10:30:45,561 ERROR [io.qua.dep.dev.IsolatedDevModeMain] (main) Failed to start quarkus: java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
	[error]: Build step io.quarkiverse.logging.logback.deployment.LoggingLogbackProcessor#removeSlf4jBinding threw an exception: java.lang.NoSuchMethodError: io.quarkus.deployment.builditem.RemovedResourceBuildItem.<init>(Lio/quarkus/bootstrap/model/AppArtifactKey;Ljava/util/Set;)V
	at io.quarkiverse.logging.logback.deployment.LoggingLogbackProcessor.removeSlf4jBinding(LoggingLogbackProcessor.java:70)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:887)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
	at java.base/java.lang.Thread.run(Thread.java:834)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)

	at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment(AugmentActionImpl.java:330)
	at io.quarkus.runner.bootstrap.AugmentActionImpl.createInitialRuntimeApplication(AugmentActionImpl.java:252)
	at io.quarkus.runner.bootstrap.AugmentActionImpl.createInitialRuntimeApplication(AugmentActionImpl.java:60)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.firstStart(IsolatedDevModeMain.java:93)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.accept(IsolatedDevModeMain.java:450)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.accept(IsolatedDevModeMain.java:67)
	at io.quarkus.bootstrap.app.CuratedApplication.runInCl(CuratedApplication.java:149)
	at io.quarkus.bootstrap.app.CuratedApplication.runInAugmentClassLoader(CuratedApplication.java:105)
	at io.quarkus.deployment.dev.DevModeMain.start(DevModeMain.java:145)
	at io.quarkus.deployment.dev.DevModeMain.main(DevModeMain.java:63)
Caused by: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
	[error]: Build step io.quarkiverse.logging.logback.deployment.LoggingLogbackProcessor#removeSlf4jBinding threw an exception: java.lang.NoSuchMethodError: io.quarkus.deployment.builditem.RemovedResourceBuildItem.<init>(Lio/quarkus/bootstrap/model/AppArtifactKey;Ljava/util/Set;)V
	at io.quarkiverse.logging.logback.deployment.LoggingLogbackProcessor.removeSlf4jBinding(LoggingLogbackProcessor.java:70)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:887)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
	at java.base/java.lang.Thread.run(Thread.java:834)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)

	at io.quarkus.builder.Execution.run(Execution.java:116)
	at io.quarkus.builder.BuildExecutionBuilder.execute(BuildExecutionBuilder.java:79)
	at io.quarkus.deployment.QuarkusAugmentor.run(QuarkusAugmentor.java:161)
	at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment(AugmentActionImpl.java:328)
	... 9 more
Caused by: java.lang.NoSuchMethodError: io.quarkus.deployment.builditem.RemovedResourceBuildItem.<init>(Lio/quarkus/bootstrap/model/AppArtifactKey;Ljava/util/Set;)V
	at io.quarkiverse.logging.logback.deployment.LoggingLogbackProcessor.removeSlf4jBinding(LoggingLogbackProcessor.java:70)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:887)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
	at java.base/java.lang.Thread.run(Thread.java:834)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
```

**Comment by stuartwdouglas** [comment]
I have released 0.6.0: https://github.com/quarkiverse/quarkus-logging-logback/pull/24

Might take a while to get to central, but should be good to go in a few hours.

**Comment by stuartwdouglas** [comment]
Oops, I messed up the 0.6.0 release, so have released 0.7.0

**Comment by eriknyk** [comment]
yeap! it works pretty good! thank you @stuartwdouglas
